Examples/UIExplorer/UIExplorer/AppDelegate.m

```diff
@@ -36,7 +36,23 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
    * on the same Wi-Fi network.
    */
 
-  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/Examples/UIExplorer/UIExplorerApp.includeRequire.runModule.bundle?dev=true"];
+  NSMutableString* url;
+  NSDictionary *dict = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"RCTWebSocketExecutor"];
```

Comment:
@frantic: Do you have any suggestions for a good place? Where do you guys put utility functions? This one?: `./React/Base/RCTUtils.m`